### PR TITLE
Configure timeline api to accept multiple words

### PR DESCRIPTION
**Pull request by Zimbardoex**
### Solves:
* #55

### CODE REVIEW

File: back-end/app/timeline/routes.py [hunk 1]
```diff
@@ -28,6 +28,49 @@
 )
 
 
+def getWordData(words, year_from, year_to):
```
Comment (NICHTJ3) [inline back-end/app/timeline/routes.py]: This should probably be named getWordFrequencyData as this file will potentially contain other routes

File: back-end/app/timeline/routes.py [hunk 2]
```diff
@@ -28,6 +28,49 @@
...
+                wordCount = model.wv.vocab[word].count
+                wordFreq = (wordCount / model.totalWordCount) * 100
+                wordRank = model.wv.vocab[word].index + 1
+                addToData(data, word, year, wordCount, wordFreq, wordRank)
+        except KeyError:  # The word is not in vocabulary for this year.
+            addToData(data, word, year)
+    return data
+
+
+def addToData(data, word, year, wordCount=0, wordFreq=0, rank=0):
```
Comment (NICHTJ3) [inline back-end/app/timeline/routes.py]: Maybe rename this to add to word frequency data?

File: back-end/app/timeline/routes.py [hunk 3]
```diff
@@ -37,36 +80,19 @@ def index():
 def getFrequency():
     try:
         # pylint: disable=unexpected-keyword-arg
-        FrequencySchema(strict=True).load(request.get_json(force=True)) if int(
+        FrequencySchema(strict=True).load(
```
Comment (NICHTJ3) [inline back-end/app/timeline/routes.py]: Nah i think it can be removed

File: back-end/app/timeline/routes.py (hunk 1 above)
Comment (NICHTJ3) [reply back-end/app/timeline/routes.py]: or just get frequency data

File: back-end/app/timeline/test_routes.py [hunk 4]
```diff
@@ -35,30 +35,30 @@ def test_frequecy_should_only_return_data_for_given_years(self):
...
     def test_should_return_years_invalid_if_year_from_after_year_to(self):
         year_from = 2018
         year_to = 2017
         response = self.client.post(
             self.prefix + "frequency",
-            json={"word": "The", "year_from": year_from, "year_to": year_to},
+            json={"words": ["The"], "year_from": year_from, "year_to": year_to},
         ).get_json()
 
         assert "year_from" in response["messages"]
```
Comment (NICHTJ3) [inline back-end/app/timeline/test_routes.py]: There should be a test here for returning multiple words

File: back-end/app/timeline/routes.py (hunk 1 above)
Comment (Zimbardoex) [reply back-end/app/timeline/routes.py]: Renamed.

File: back-end/app/timeline/routes.py (hunk 2 above)
Comment (Zimbardoex) [reply back-end/app/timeline/routes.py]: Refactored and added functionality to getWordFrequencyData

File: back-end/app/timeline/routes.py (hunk 3 above)
Comment (Zimbardoex) [reply back-end/app/timeline/routes.py]: Will create separate issue.

File: back-end/app/timeline/test_routes.py (hunk 4 above)
Comment (Zimbardoex) [reply back-end/app/timeline/test_routes.py]: Test added.

File: back-end/app/timeline/routes.py [hunk 5]
```diff
@@ -14,19 +15,69 @@
 
 # TODO: Tidy this up as it seems like bad practice
 # Use models from enviroment if not testing
+testModel = Word2Vec(
```
Comment (NICHTJ3) [inline back-end/app/timeline/routes.py]: Didn't realize i missed this

**Pull request merged**